src/Rings/groebner.jl

```diff
@@ -1484,8 +1484,7 @@ end
 
 # modular gröbner basis techniques using Singular
 @doc raw"""
-groebner_basis_modular(I::MPolyIdeal{fmpq_mpoly}; ordering::MonomialOrdering = default_ordering(base_ring(I)),
-                           certify::Bool = false)
+  groebner_basis_modular(I::MPolyIdeal{fmpq_mpoly}; ordering::MonomialOrdering = default_ordering(base_ring(I)), certify::Bool = false)
```

Comment:
```suggestion
    groebner_basis_modular(I::MPolyIdeal{fmpq_mpoly}; ordering::MonomialOrdering = default_ordering(base_ring(I)), certify::Bool = false)
```
here 4 spaces are needed